docs/content/guide/providers.ngdoc

```diff
@@ -212,7 +212,7 @@ that need it. We can make it configurable like so:
 
 
 ```javascript
-myApp.provider('unicornLauncher', function UnicornLauncherProvider() {
+myApp.provider('unicornLauncherProvider', function UnicornLauncherProvider() {
```

Comment:
This is not right. This is registering a provider for the `unicornLauncher` service. The service will be injectable as `unicornLauncher`. (The provider itself will be injectable as `unicornLauncherProvider` during the config phase, but that is irrelevant.)